module/metrics/execution.go

```diff
@@ -739,6 +749,11 @@ func (ec *ExecutionCollector) ExecutionTransactionExecuted(
 	ec.transactionExecutionTime.Observe(float64(dur.Milliseconds()))
 	ec.transactionConflictRetries.Observe(float64(numConflictRetries))
 	ec.transactionComputationUsed.Observe(float64(compUsed))
+	if compUsed > 0 {
+		// normalize so the value should be around 1
+		ec.transactionNormalizedTimePerComputation.Observe(
+			(dur.Seconds() / float64(compUsed)) * flow.EstimatedComputationPerMillisecond)
```

Comment:
I don't understand the math here. And also why using `Seconds()` instead of `Millisecond`?

Reply:
good catch!